common/src/storage/primitives/map.rs

```diff
@@ -21,8 +21,7 @@
 //! This primitive defines interface of interaction
 //! with globally stored single-key map (Key -> Value).
 
-use frame_support::codec::{Encode, EncodeAppend, EncodeLike};
-
+use sp_runtime::codec::{Encode, EncodeAppend, EncodeLike};
```

Comment:
here and in similar places directly from codec mb?
btw was there any reason to switch the dep source? frame_support no longer re-imports these?

Reply:
> frame_support no longer re-imports these?
Yes